test/powershell/Language/Scripting/Array.Tests.ps1

```diff
@@ -0,0 +1,54 @@
+Describe "ArrayExpression Tests" -Tags "CI" {
+    It "@([object[]](1,2,3)) should return a 3-element array of object[]" {
+        $result = @([object[]](1,2,3))
+        $result.GetType().FullName | Should Be "System.Object[]"
+        $result.Length | Should Be 3
```

Comment:
Ah, I see your point. A `Should Not BeNullOrEmpty` check should be done in that case.